docs/user_guide/analytics.rst

```diff
@@ -1,6 +1,35 @@
 Analytics and usage services
 ============================
 
+The theme supports analytics via the ``analytics`` option. It is configured
+by passing a dictionary with options. See the sections bellow for relevant
+options depending on the analytic provider that you want to use.
+
+.. code:: python
+
+   html_theme_options = {
+       "analytics": analytics_options,
+   }
+
+Plausible Analytics (recommended)
```

Comment:
I feel like we shouldn't say "recommended" in the title, given that it is not free for people who can't get a server via the scientific-python ecosystem, and people might use this theme that are not part of that community. Instead, what if we put an admonition like this at the top:

````
Generally speaking we recommend using Plausible over Google Analytics because it has a better story around user security and privacy.
In addition, it is more open-source and transparent.
In fact, `you can self-host a Plausible server <https://www.elvisduru.com/blog/how-to-self-host-plausible-analytics>`__.

.. admonition:: Get a self-hosted Plausible server at ``scientific-python.org``
   :class: tip

   If your documentation is for a package that is part of the SciPy / PyData ecosystem, they might be able to host a Plausible server for you at `<your-package>.scientific-python.org`.
   To ask about this, contact them on the social media platform of your choice and learn more at `scientific-python.org <https://scientific-python.org>`__.
````

Reply:
I will use your suggestion.